pom.xml

```diff
@@ -7,7 +7,7 @@
     <parent>
         <groupId>uk.gov.justice</groupId>
         <artifactId>parent-pom</artifactId>
-        <version>1.6.9</version>
+        <version>1.6.10-SNAPSHOT</version>
```

Comment:
Does this need to be reverted back to a released version?
But maybe we need to release the parent pom if there are changes?

Same as below.